osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -321,6 +323,9 @@ private void selectNextDifficulty(int direction)
         /// <returns>True if a selection could be made, else False.</returns>
         public bool SelectNextRandom()
         {
+            if (!AllowSelection)
+                return false;
```

Comment:
what's the reasoning for this one? removing it seems to have changed nothing in my quick test.

Reply:
I doubt it will change anything, just feels more correct.